src/vivarium_gates_nutrition_optimization/components/pregnancy.py

```diff
@@ -47,7 +127,7 @@ def Pregnancy():
             "prevalence": lambda *_: 0.0,
             "disability_weight": lambda *_: 0.0,
             "excess_mortality_rate": lambda *_: 0.0,
-            "dwell_time": lambda *_: pd.Timedelta(days=POSTPARTUM_DURATION),
+            "dwell_time": lambda *_: pd.Timedelta(days=DURATIONS.POSTPARTUM_DURATION),
```

Comment:
Nit: since the class is already called `DURATIONS`, you can shorten this to `POSTPARTUM`.